src/kernels/miopen_limits.hpp

```diff
@@ -41,6 +41,15 @@ namespace std {
 template <typename T>
 class numeric_limits;
 
+template <>
```

Comment:
@atamazov could you take a look? `test_reduce_test` seems to be needing this in the docker where I am testing everything. Also I may wonder if there could be other data types missing?